amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3911,102 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventsInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventInfo'
+  EventInfo:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+        description: this is the id of the event
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyCreate:
+    type: object
+    required:
+      - title
+    properties:
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+        default: Simple
+  EventEntry:
+    type: object
+    required:
+      - name
+      - type
+    properties:
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyUpdate:
+    type: object
+    required:
+      - id
+    properties:
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+        default: Simple
+  EventTypeInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventTypeEntry'
+  EventTypeEntry:
+    type: object
+    required:
+      - id
+      - name
+      - isAutomatic
```

Comment:
I don't think this is required because it has a default.

Reply:
Done